src/shared/components/form/formInput/formInput.js

```diff
@@ -44,14 +44,14 @@ class FormInput extends Component {
         {this.props.label && <Label htmlFor={this.props.id}>{this.props.label}</Label>}
 
         <input
-          className={!this.state.isValid && styles.error}
+          className={(!this.state.isValid && this.props.submitted) && styles.error}
           id={this.props.id}
           type={this.props.inputType}
           value={this.state.text}
           placeholder={this.props.placeholder}
           onChange={this.handleChange}
         />
-        {!this.state.isValid && <span>{this.props.validationErrorMessage}</span>}
+        {(!this.state.isValid && this.props.submitted) && <span>{this.props.validationErrorMessage}</span>}
```

Comment:
This would make `submitted` to be a required prop, breaking uses of component with null prop.  A ternary operation with a check for null is the solution here, I believe.

Reply:
Then we can apply it only for specific input field, in this case only for e-mail field by setting/checking the 'type' of the field?

Reply:
@nick-barth How does this make `submitted` a required prop? Whether the original passed-in value of `this.props.submitted` is `null` or `false`, the second part of the `&&` conditional will evaluate to false, and the validation error message will not display.  That seems to correctly match the intent of the bugfix.